### PR TITLE
HAAS-289

### PR DESCRIPTION
**Pull request by mjga-trifork**
Added exactly requirements for consent.provision and consent.provision.type and changed the property.valueString in smart scopes to conform to standard.
http://hl7.org/fhir/smart-app-launch/scopes-and-launch-context.html#clinical-scope-syntax

### CODE REVIEW

File: input/fsh/codesystems/smart-scopes.fsh [hunk 1]
```diff
@@ -26,7 +27,7 @@ Description: "SMART Scope codes for use in CompossanaConsent"
 * #avmguidance ^designation[+].language = #fr-CH
 * #avmguidance ^designation[=].value = "translation needed"
 * #avmguidance ^property.code = #smart-scope
-* #avmguidance ^property.valueString = "avmguidance/*.rs"
+* #avmguidance ^property.valueString = "user/avmguidance.rs"
```
Comment (jkiddo) [inline input/fsh/codesystems/smart-scopes.fsh:30]: There is no resource called 'avmguidance'

File: input/fsh/codesystems/smart-scopes.fsh [hunk 2]
```diff
@@ -38,7 +39,7 @@ Description: "SMART Scope codes for use in CompossanaConsent"
 * #clinicalDocument ^designation[+].language = #fr-CH
 * #clinicalDocument ^designation[=].value = "translation needed"
 * #clinicalDocument ^property.code = #smart-scope
-* #clinicalDocument ^property.valueString = "clinicalDocument/*.crs"
+* #clinicalDocument ^property.valueString = "user/clinicalDocument.crs"
```
Comment (jkiddo) [inline input/fsh/codesystems/smart-scopes.fsh:42]: this does not follow the smart spec

File: input/fsh/codesystems/smart-scopes.fsh [hunk 3]
```diff
@@ -74,7 +75,7 @@ Description: "SMART Scope codes for use in CompossanaConsent"
 * #personalData ^designation[+].language = #fr-CH
 * #personalData ^designation[=].value = "translation needed"
 * #personalData ^property.code = #smart-scope
-* #personalData ^property.valueString = "personalData/*.r"
+* #personalData ^property.valueString = "user/personalData.r"
```
Comment (jkiddo) [inline input/fsh/codesystems/smart-scopes.fsh:78]: this does not follow the smart spec

**Review by jkiddo — CHANGES_REQUESTED** [review]

File: input/fsh/codesystems/smart-scopes.fsh (hunk 1 above)
Comment (mjga-trifork) [reply input/fsh/codesystems/smart-scopes.fsh:30]: Of course. We need to identify or define Fhir resource types for use her. I guess avmguidance is not a standard fhir resource.

Comment (mjga-trifork) [reply input/fsh/codesystems/smart-scopes.fsh]: I've generalized all these to wildcards

File: input/fsh/codesystems/smart-scopes.fsh (hunk 3 above)
Comment (mjga-trifork) [reply input/fsh/codesystems/smart-scopes.fsh]: I've generalized all these to wildcards

File: input/fsh/codesystems/smart-scopes.fsh (hunk 2 above)
Comment (mjga-trifork) [reply input/fsh/codesystems/smart-scopes.fsh]: I've generalized all these to wildcards

**Pull request merged**